tests/test_wave_05.py

```diff
@@ -71,29 +70,33 @@ def test_create_goal(client):
         }
     }
 
-@pytest.mark.skip(reason="test to be completed by student")
+
 def test_update_goal(client, one_goal):
-    pass
     # Act
     # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json={
+        "title": "Updated Goal Title",
+    })
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
```

Comment:
We also want assertions to check that the `response_body` is what we're looking for.